lesson-4/JAMROZ-week 4 homework/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <!--Metdata goes here-->
+    <title>My Portfolio</title>
+  </head>
+  <link rel="stylesheet" href="style.css"/>
+  <body>
+
+    <!--Content goes here-->
+    <h1>My Portfolio</h1>
+
+    <p>My name is Kenna Jamroz, and I'm a CAMD student at <em>Northeastern University.</em> I want to use this space to showcase the pieces I've been working on recently both in and outside of class.</p>
+
+    <h2>Projects</h2>
+
+
+    <a href="Paintings/" class="paintingsbutton">Paintings<a/>
```

Comment:
It's ok for this, but since you named your folder `paintings` you should write this URL as `paintings/` in all lowercase